docs/02-sygma-sdk/03-Quick-Start/02-transfer-token.md

```diff
@@ -7,4 +7,56 @@ sidebar_position: 2
 draft: true
 ---
 
-More to come soon...
\ No newline at end of file
+Transferring a token with `Sygma SDK` is very straigthforward. You need to build the transfer object to pass it to your signer to send the transaction.
+
+```ts
+async function buildTransfer(
+  wallet: Wallet,
+  from: Domain,
+  to: Domain,
+  resource: Resource,
+  recipient: string,
+  amount: string
+): Promise<Transfer<Fungible>> {
+  return {
+    sender: await wallet.getAddress(),
+    amount: {
+      amount,
+    },
+    from,
+    to,
+    resource,
+    recipient,
+  };
+};
+// use the function
+const transfer = await buildTransfer(wallet, from ,to, resource, recipient, amount);
+```
+
+You will also need to get the current `fee`. For this is as simple as doing the following:
+
+```ts
+const assetTransfer = new EVMAssetTransfer();
+await assetTransfer.init(provider, Environment.DEVNET);
+const fee = await assetTransfer.getFee()
+```
+
```

Comment:
I would here add also approvals preparation, with simple note - something like this:

You can check if approvals are required for your transfer.
```
  const approvals = await assetTransfer.buildApprovals(transfer, fee);
  if (approvals.len > 0) {
    // sign and send approvals
  }
```

Reply:
Done